projects/public-search/src/app/patron-profile/patron-profile-password/patron-profile-password.component.html

```diff
@@ -0,0 +1,32 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2022 RERO
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="container pl-0">
+  <form [formGroup]="form" (ngSubmit)="submit()">
+    <div class="row">
+      <formly-form class="col mb-0" [model]="model" [fields]="fields" [form]="form"></formly-form>
+    </div>
+    <div class="col pl-0">
+      <button type="button" id="editor-delete-button" class="btn btn-outline-danger btn-sm mr-1"
+          (click)="cancel()">
+          <i class="fa fa-times"></i> {{ 'Cancel' | translate }}
+        </button>
+        <button type="submit" id="editor-save-button" class="btn btn-primary btn-sm" [disabled]="!form.valid">
+          <i class="fa fa-save"></i> {{ 'Change Password' | translate }}
+        </button>
+      </div>
+    </form>
+</div>
```

Comment:
COSMETIC : `container`, `row` and `col` could be removed, it shouldn't change the UI aspect (maybe keep `container` for "max-width css rule")